crates/redstone/src/crypto/mod.rs

```diff
@@ -47,13 +50,21 @@ pub trait Crypto {
     }
 }
 
+fn check_signature_malleability(sig: &[u8]) -> Result<(), CryptoError> {
+    if U256::from_big_endian(&sig[32..64]) > ECDSA_N_DIV_2.into() {
```

Comment:
Hmmm, [U256 ](https://docs.rs/primitive-types/latest/primitive_types/struct.U256.html#) have pub LE representation.  Can we rewrite ECDSA_N_DIV_2 to be constant from [u64; 4]?

Reply:
I did [u8;32] as it is the closest we can get to construct U256 ( by using `from_be_bytes`). This shall have much smaller overhead than from hex `str`.